example/spec/discoverAndQueuePdrs/DiscoverAndQueuePdrsSuccessSpec.js

```diff
@@ -46,17 +57,68 @@ describe('The Discover And Queue PDRs workflow', () => {
   });
 
   describe('the QueuePdrs Lambda', () => {
-    let lambdaOutput = null;
+    it('has expected output', () => {
+      expect(queuePdrsOutput.payload.pdrs_queued).toEqual(1);
+      expect(queuePdrsOutput.payload.running.length).toEqual(1);
+    });
+  });
+
+  /**
+   * The DiscoverAndQueuePdrs workflow kicks off a ParsePdr workflow, so check that the
+   * ParsePdr workflow completes successfully. Above, we checked that there is
+   * one running task, which is the ParsePdr workflow. The payload has the arn of the
+   * running workflow, so use that to get the status.
+   */
+  describe('ParsePdr workflow', () => {
+    let parsePdrWorkflowArn;
+    let parsePdrExecutionStatus;
 
     beforeAll(async () => {
-      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'QueuePdrs');
+      parsePdrWorkflowArn = queuePdrsOutput.payload.running[0];
+      parsePdrExecutionStatus = await waitForCompletedExecution(parsePdrWorkflowArn);
+    });
+
+    it('executes successfully', () => {
+      expect(parsePdrExecutionStatus).toEqual('SUCCEEDED');
     });
 
-    it('output is pdrs_queued', () => {
-      expect(lambdaOutput.payload).toEqual({ pdrs_queued: 1 });
+    describe('ParsePdr lambda function', () => {
+      it('outputs 1 granule and pdr', async () => {
```

Comment:
I would change the language here to something like: successfully parses a granule from the PDR